### PR TITLE
Bumping the version number to 1.0.0

### PR DESCRIPTION
**Pull request by arsen-es**
*Issue #, if available:* N/A

*Description of changes:* Version upgrade

*How tested:* successful build, plus connected to local es cluster and browsed the indexes with SQuirreL.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: opendistro-elasticsearch-jdbc.release-notes [hunk 1]
```diff
@@ -1,5 +1,11 @@
+## 2019-06-06, Version 1.0.0 (Current)
 
-## 2019-04-19, Version 0.9.0 (Current)
+### Changes
+
+* Update to LocalDateTime conversion to Timestamp to support timezone [issue #6]
```
Comment (qreshi) [inline opendistro-elasticsearch-jdbc.release-notes]: NP: Rewording to "Update```d``` to LocalDateTime conversion ```for``` Timestamp to support timezone" may be better here

Comment (arsen-es) [reply opendistro-elasticsearch-jdbc.release-notes]: I initially had "Updated the ... conversion" but then saw the previous entries were nouns, not verbs, so changed to noun.

Comment (qreshi) [reply opendistro-elasticsearch-jdbc.release-notes]: I noticed the one below it says "Updated"

Comment (arsen-es) [reply opendistro-elasticsearch-jdbc.release-notes]: :)) You caught me. So which one do you want me to change? Does it even matter?

Comment (qreshi) [reply opendistro-elasticsearch-jdbc.release-notes]: Either is fine, no preference on my end :)

**Review by qreshi — APPROVED** [review]

**Pull request merged**